session/session.go

```diff
@@ -1469,21 +1469,13 @@ func (s *session) ParseWithParams(ctx context.Context, sql string, args ...inter
 		}
 	}
 
-	internal := s.isInternal()
```

Comment:
so, this means there will be **NO** user SQL run here?

Reply:
Yes. But after discuss with @xhebox, for scalability, I will add an interface called `ParseWithParamsInternal`.

Reply:
What is the purpose to introduce `ParseWithParamsInternal`? How is it different from `ParseWithParams`?

Reply:
`Parse()` is deprecated, it will be insteaded by `ParseWithParams` which should have ability to deal with users' SQLs. So add `ParseWithParamsInternal` to process internal SQLs. @tangenta